lib/players/human.rb

```diff
@@ -0,0 +1,24 @@
+module Players
+    class Human < Player
+        
+        attr_accessor:input
+        
+        def receive_input
```

Comment:
I definitely see the problem here, but I'm having trouble implementing the solution. Whenever I move the "input = gets.strip" line to the CLI class, the program works just fine when I run it via bin/tictactoe. However, when I try to run the test suite, rspec hangs when it hits the tests involving the Players::Human class. The tests don't fail, rspec just hangs until I ctrl+C out of it.

Currently I have two different #move methods - one that passes spec and one that works when I run bin/tictactoe. Any ideas on how I can resolve the rspec issue?